temoa/cli.py

```
@@ -0,0 +1,286 @@
+from temoa.core.modes import TemoaMode
+from temoa.version_information import TEMOA_MAJOR, TEMOA_MINOR
+
+# =============================================================================
+# Logging & Helper Setup
+# =============================================================================
+logger = logging.getLogger(__name__)
+
+
+def _create_output_folder() -> Path:
+    """Create a default time-stamped folder for outputs."""
+    output_path = Path('output_files', datetime.now().strftime('%Y-%m-%d_%H%M%S'))
```

Ruff (0.14.3)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)